ganttproject/src/net/sourceforge/ganttproject/client/RssParser.java

```diff
@@ -84,6 +85,18 @@ public RssFeed parse(InputStream inputStream, Date lastCheckDate) {
     return result;
   }
 
+  public RssUpdate parseUpdate(String content){
+    if(!StringUtils.isEmptyOrNull(content)) {
+      content = content.replaceAll("</?div>(\\n)?", "");
```

Comment:
It is not that easy because, although the content is HTML, it is 1) escaped HTML and 2) there are attributes inside div

With respect to 1), you can unescape the content with `com.google.common.xml.XmlEscapers#xmlAttributeEscaper`.

With respect to 2), it seems that we'll have to build this feed on our own anyway, so we'll filter out the wrapping `div`. You may think that after unescaping it will be exactly like 
```
VERSION
LINK
SOME HTML
```

You can find an example  here: https://www.dropbox.com/s/6dnpdw3evhddcad/update.rss?dl=1 
It is a copy of our current feed plus update entry which is formatted in the way described above.

You can temporarily replace net.sourceforge.ganttproject.client.RssFeedChecker#RSS_URL value with that URL

Reply:
com.google.common.xml.XmlEscapers#xmlAttributeEscaper. escape also html tags inside the string
Maybe I should just remove wrapped div but not with regex that may also remove other div tags?

Reply:
You need to unescape html tags anyway to pass HTML to the editor pane.